scripts/us_cdc/500_places/parse_cdc_places.py

```diff
@@ -0,0 +1,349 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+Author: Padma Gundapaneni @padma-g
+Date: 8/19/2021
+Description: This script cleans a csv file of census tract, county,
+or city level CDC 500 PLACES data downloaded from the CDC.
+URL: https://chronicdata.cdc.gov/browse?category=500+Cities+%26+Places
+@input_file     filepath to the original csv that needs to be cleaned
+@output_file    filepath to the csv to which the cleaned data is written
+python3 parse_cdc_places.py input_file output_file
+'''
+
+import sys
+import pandas as pd
+import numpy as np
+
+# Mapping of measure abbreviations to StatVar dcids
+MEASURE_TO_STATVAR_MAP = {
+    "TEETHLOST":
+        "Percent_Person_65OrMoreYears_WithAllTeethLoss",
+    "MAMMOUSE":
+        "Percent_Person_50To74Years_Female_ReceivedMammography",
+    "ARTHRITIS":
+        "Percent_Person_WithArthritis",
+    "STROKE":
+        "Percent_Person_WithStroke",
+    "OBESITY":
+        "Percent_Person_Obesity",
+    "CASTHMA":
+        "Percent_Person_WithAsthma",
+    "DIABETES":
+        "Percent_Person_WithDiabetes",
+    "BINGE":
+        "Percent_Person_BingeDrinking",
+    "CSMOKING":
+        "Percent_Person_Smoking",
+    "SLEEP":
+        "Percent_Person_SleepLessThan7Hours",
+    "COLON_SCREEN":
+        "Percent_Person_50To75Years_ReceivedColorectalCancerScreening",
+    "CANCER":
+        "Percent_Person_WithCancerExcludingSkinCancer",
+    "PHLTH":
+        "Percent_Person_WithPhysicalHealthNotGood",
+    "COREM":
+        "Percent_Person_65OrMoreYears_Male_ReceivedCorePreventiveServices",
+    "COREW":
+        "Percent_Person_65OrMoreYears_Female_ReceivedCorePreventiveServices",
+    "CHECKUP":
+        "Percent_Person_ReceivedAnnualCheckup",
+    "BPHIGH":
+        "Percent_Person_WithHighBloodPressure",
+    "DENTAL":
+        "Percent_Person_ReceivedDentalVisit",
+    "MHLTH":
+        "Percent_Person_WithMentalHealthNotGood",
+    "LPA":
+        "Percent_Person_PhysicalInactivity",
+    "ACCESS2":
+        "Percent_Person_18To64Years_ReceivedNoHealthInsurance",
+    "COPD":
+        "Percent_Person_WithChronicObstructivePulmonaryDisease",
+    "CHOLSCREEN":
+        "Percent_Person_ReceivedCholesterolScreening",
+    "KIDNEY":
+        "Percent_Person_WithChronicKidneyDisease",
+    "HIGHCHOL":
+        "Percent_Person_WithHighCholesterol",
+    "BPMED":
+        "Percent_Person_18OrMoreYears_WithHighBloodPressure_ReceivedTakingBloodPressureMedication",
+    "CHD":
+        "Percent_Person_WithCoronaryHeartDisease",
+    "CERVICAL":
+        "Percent_Person_21To65Years_Female_ReceivedCervicalCancerScreening"
+}
+
+# Mapping of StatVar dcids to StatVar names
+STATVAR_NAMES_MAP = {
```

Comment:
Is this used?  Generally you can rely on the the names being auto-generated based on the StatVar definition

Reply:
Forgot to remove this, thanks!